stable/distribution/templates/_helpers.tpl

```diff
@@ -69,11 +69,5 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-{{/*
-Expand the name of the chart.
-*/}}
-{{- define "redis.name" -}}
-{{- default .Chart.Name .Values.redis.name | trunc 63 | trimSuffix "-" -}}
```

Comment:
@chukka Please delete `redis.name` from the values.yaml

Reply:
Done